Tests/MapboxMapsTests/Location/LocationManagerTests.swift

```diff
@@ -151,4 +154,16 @@ final class LocationManagerTests: XCTestCase {
         XCTAssertTrue(delegate.didChangeAccuracyAuthorizationStub.invocations.first?.parameters.locationManager === locationManager)
         XCTAssertEqual(delegate.didChangeAccuracyAuthorizationStub.invocations.first?.parameters.accuracyAuthorization, accuracyAuthorization)
     }
+
+    func testAddPuckLocationConsumer() {
```

Comment:
Might be a good idea to test that the location consumers are getting notified when interpolated location is updated.

Reply:
good idea. I have added new test case for this